packages/cli/src/formatter.ts

```diff
@@ -523,6 +523,27 @@ export const formatConfiguredServices = (serviceNames: ReadonlyArray<string>): s
   return formattedServices.join('\n')
 }
 
+const privateAdapters = ['dummy']
+
+export const getPrivateAdaptersNames = (): string[] => privateAdapters
```

Comment:
tiny nit:
making `privateAdapters` a global constant and then returning it as a read-write array opens the possibility of modifying the list of private adapters in runtime by accident.
this could be a little mitigated by returning a `ReadonlyArray<string>` instead of `string[]` but more generally it would be good practice to avoid having global context in the first place, so I'd suggest it would be safer to create the list in the function each time it is called

```suggestion
export const getPrivateAdaptersNames = (): ReadonlyArray<string> => ['dummy']
```